src/composite.jl

```diff
@@ -0,0 +1,54 @@
+"""
+   Composite <: Transform
+
+A `Composite` transform is a composition of `Transform`s, currently limited to `OneToOne()`
+cardinality. It can be fit and applied in a single step.
+
+The transforms in `Composite([t1, t2, t3])` are applied in `t1`, `t2`, `t3` order, where
+the output of `t1` is the input to `t2` etc. When using `∘` to create transforms, the order
+is `t3 ∘ t2 ∘ t1`, as in function composition.
```

Comment:
hmm...that might get confusing...

Reply:
Yeah I had it the other way around first but the issue then is that the `c.transforms[1]` is the last transform that is applied, which is even more confusing I think.

The only totally non-confusing way is to get rid of the ` ∘ ` syntactic sugar, which is what switches the order.

Reply:
>The only totally non-confusing way is to get rid of the ∘ syntactic sugar, which is what switches the order.

I wouldn't be against this tbh, but it might be handy for building pipelines of transforms like
```julia
tc = Composite(t1, t2)
...
...
tc = t3 ∘ tc
...
tc = t4 ∘ tc
```

so I can see value in it being used that way